sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -1196,16 +1196,16 @@ def test_gradient_boosting_early_stopping():
 
     # Without early stopping
     gbc = GradientBoostingClassifier(
-        n_estimators=100, learning_rate=0.1, max_depth=3, random_state=42
+        n_estimators=5, learning_rate=0.1, max_depth=3, random_state=42
     )
     gbc.fit(X, y)
     gbr = GradientBoostingRegressor(
-        n_estimators=200, learning_rate=0.1, max_depth=3, random_state=42
+        n_estimators=10, learning_rate=0.1, max_depth=3, random_state=42
```

Comment:
Same remark here, we need significantly more than 13. Let's use 30 for instance:

```suggestion
        n_estimators=30, learning_rate=0.1, max_depth=3, random_state=42
```